api/spec/src/subscription.tsp

```diff
@@ -0,0 +1,266 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+import "./types.tsp";
+import "./ratecards.tsp";
+import "./plan.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter;
+
+@route("/api/v1/subscriptions")
+@tag("Subscriptions")
+interface Subscriptions {
+  @get
+  @operationId("getSubscription")
+  get(@path subscriptionId: ULID): Subscription | OpenMeter.NotFoundError | OpenMeter.CommonErrors;
+
+  create(
+    @body body: {
+      customerId: ULID;
+      templatingPlan: {
+        planKey: Key;
+        planVersion: integer;
+      };
+      currency: CurrencyCode;
+      activeFrom: DateTime;
```

Comment:
`activeSince` or `startDate`?

Reply:
For subscription in particular  `activeFrom` & `activeTo` doesnt sound the best, but i'd like them to be the standard across resources, see `CadencedModel`. (I'm open to other suggestions for the standard but I suppose that's the most general term we can use)